specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/managedHsm.json

```diff
@@ -0,0 +1,1609 @@
+          "description": "Purge protection status of the original managed HSM."
+        },
+        "tags": {
+          "readOnly": true,
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags of the original managed HSM."
+        }
+      },
+      "description": "Properties of the deleted managed HSM.",
+      "type": "object"
+    },
+    "DeletedManagedHsmListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedManagedHsm"
+          },
+          "description": "The list of deleted managed HSM Pools."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of deleted managed HSM Pools."
+        }
+      },
+      "description": "List of deleted managed HSM Pools",
+      "type": "object"
+    },
+    "MHSMNetworkRuleSet": {
+      "properties": {
+        "bypass": {
+          "type": "string",
+          "description": "Tells what traffic can bypass network rules. This can be 'AzureServices' or 'None'.  If not specified the default is 'AzureServices'.",
+          "enum": [
+            "AzureServices",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "NetworkRuleBypassOptions",
+            "modelAsString": true
+          }
+        },
+        "defaultAction": {
+          "type": "string",
+          "description": "The default action when no rule from ipRules and from virtualNetworkRules match. This is only used after the bypass property has been evaluated.",
+          "enum": [
+            "Allow",
+            "Deny"
+          ],
+          "x-ms-enum": {
+            "name": "NetworkRuleAction",
+            "modelAsString": true
+          }
+        },
+        "ipRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MHSMIPRule"
+          },
+          "x-ms-identifiers": [
+            "value"
+          ],
+          "description": "The list of IP address rules."
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MHSMVirtualNetworkRule"
+          },
+          "description": "The list of virtual network rules."
+        }
+      },
+      "description": "A set of rules governing the network accessibility of a managed hsm pool.",
+      "type": "object"
+    },
+    "MHSMIPRule": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "An IPv4 address range in CIDR notation, such as '124.56.78.91' (simple IP address) or '124.56.78.0/24' (all addresses that start with 124.56.78)."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "A rule governing the accessibility of a managed hsm pool from a specific ip address or ip range.",
```

Comment:
>d hsm pool from a specific ip address or ip range.", [](http://example.com/codeflow?start=68&length=52)

nit, we should capitalize it as IP address or IP range in docs

Reply:
Done.